app/controllers/users/backup_code_setup_controller.rb

```diff
@@ -30,7 +30,9 @@ def create
       track_backup_codes_created
     end
 
-    def edit; end
+    def edit
+      analytics.backup_code_regenerate_visit
```

Comment:
This screen can be visited in the account creation flow. Per the AC of the ticket, can we ensure that we have a way to differentiate whether the user is regenerating backup codes from the account dashboard vs. account creation?

**Account Creation:**

1. Go to http://localhost:3000
2. Click Create an account
3. Create an account up to MFA selection
4. Choose Backup codes
5. Continue up to "You’ve added your first authentication method!" screen
6. Click "Skip for now"
7. Click "I need new backup codes"
8. This event would be logged, and I should be able to query distinct from existing account

**Existing Account:**

1. Go to http://localhost:3000
2. Sign in with an existing account
3. From the account dashboard, click "Get backup codes"
4. This event would be logged, and I should be able to query distinct from account creation

Reply:
That should now be attached to the event by way of 
`{"event_properties":{"request_came_from":"accounts#show"}...}`